### PR TITLE
add utility method valuesGradle()

### PR DESCRIPTION
**Pull request by sdelamo**

### CODE REVIEW

File: starter-core/src/test/groovy/io/micronaut/starter/options/BuildToolSpec.groovy [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+import spock.lang.Specification
+
+class BuildToolSpec extends Specification {
+
+    void "valueGradle return Groovy and Kotlin DSL"() {
+        BuildTool.valuesGradle().stream().allMatch { it.isGradle() }
+        BuildTool.valuesGradle().stream().anyMatch { BuildTool.GRADLE_KOTLIN }
+        BuildTool.valuesGradle().stream().anyMatch { BuildTool.GRADLE }
+        [BuildTool.GRADLE, BuildTool.GRADLE_KOTLIN].size() == BuildTool.valuesGradle().size()
+    }
```
Comment (timyates) [inline starter-core/src/test/groovy/io/micronaut/starter/options/BuildToolSpec.groovy]:
```suggestion
    void "valueGradle return Groovy and Kotlin DSL"() {
        expect:
        BuildTool.valuesGradle().stream().allMatch { it.isGradle() }
        BuildTool.valuesGradle().stream().anyMatch { BuildTool.GRADLE_KOTLIN }
        BuildTool.valuesGradle().stream().anyMatch { BuildTool.GRADLE }
        [BuildTool.GRADLE, BuildTool.GRADLE_KOTLIN].size() == BuildTool.valuesGradle().size()
    }
```

Comment (timyates) [reply starter-core/src/test/groovy/io/micronaut/starter/options/BuildToolSpec.groovy]: Otherwise, this isn't run as a test

**Review by timyates — CHANGES_REQUESTED** [review]

File: starter-core/src/test/groovy/io/micronaut/starter/options/BuildToolSpec.groovy (hunk 1 above)
Comment (timyates) [reply starter-core/src/test/groovy/io/micronaut/starter/options/BuildToolSpec.groovy]: Fixed in https://github.com/micronaut-projects/micronaut-starter/pull/1657/commits/dfddd837e65952b260aaf4838bca0a2d70b81424

**Review by timyates — APPROVED** [review]

File: starter-core/src/test/groovy/io/micronaut/starter/options/BuildToolSpec.groovy (hunk 1 above)
Comment (sdelamo) [reply starter-core/src/test/groovy/io/micronaut/starter/options/BuildToolSpec.groovy]: good catch

**Pull request merged**